docs/.rubocop.yml

```diff
@@ -0,0 +1,25 @@
+inherit_from: ../Library/.rubocop.yml
+
+AllCops:
+  Exclude:
+  - Gemfile
+  - ".mdl*.rb"
+  - Rakefile
+  - "_site/**/*"
+
+# These are included in docs deliberately to show what
+# `brew create` does and what the user should replace.
+FormulaAudit/Comments:
+  Enabled: false
+
+# The bottle block line length is long in its full form.
```

Comment:
Would be better to exclude bottle blocks instead of bumping line length globally